### ISSUE TITLE
Menu: keep menu open on mouseout

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#92]
Consensus seems to be that menu (and select) should not auto-collapse on mouseout, but only when clicking elsewhere on page.

It would appear that the current makeup-expander autoCollapse option would need slight modification to support this. Here it is currently:

`options.autoCollapse`: whether click-activated expander should collapse when keyboard or mouse leaves the content or widget (default: false)

We probably want to continue to with the auto collapse when keyboard leaves the widget, but not when mouse leaves the widget.

**Comment by yomed** [comment]
I looked into this a bit. I tried to simply copy/paste `autoCollapse` functionality, but with removing the `mouseleave` behavior from this:
https://github.com/makeup-js/makeup-expander/blob/a82886b3d1975d22ca3ab1df92bd19d2e877f27e/src/index.js#L79-L93

But the issue I ran into is that it seems like `focusExit` fires when focus goes on to another element, but it does not get fired if the focus just leaves the current element without going anywhere in particular (which would often be the case for clicking outside of the menu to close it). Here's that part: https://github.com/makeup-js/makeup-exit-emitter/blob/463ba1e457b077cc20c33e8d09113433cbceee03/index.js#L20-L35

Since this impacts a bunch of makeup plugins whose changes could have lots of consequences, it's probably best to leave this to you, Ian. Otherwise, let me know if you think I'm missing something or overthinking this.

**Comment by ianmcburnie** [comment]
Yeah, I'm thinking I need to add two new options, `autoKeyboardCollapse` and `autoMouseCollapse`. For backwards compatibility I could just leave in the current `autoCollapse` functionality which basically sets both of the above to true.
